change/@fluentui-react-breadcrumb-preview-562a5dd1-ef15-4eec-9351-0a1ce84c026d.json

```diff
@@ -0,0 +1,7 @@
+{
```

Comment:
## [🕵 fluentuiv9 Open the Visual Regressions report to inspect the affected screenshots]( https://staticpagerelease.z16.web.core.windows.net/azurestaticapps/vr-approval/index.html?organization=uifabric&project=fabricpublic&prId=28883&commitId=dac1be9239914f84a116a689a95e772935e735b8&env=prod&clientType=FLUENTUI)


  <details><summary><b>Breadcrumb Converged</b> 8 screenshots</summary>

|Image Name|Diff(in Pixels)| Image Type
|-|-|-|
|Breadcrumb Converged.divider type.hover.chromium.png|374| Changed|
|Breadcrumb Converged.size.hover.chromium.png|372| Changed|
|Breadcrumb Converged.size.default.chromium.png|372| Changed|
|Breadcrumb Converged.size.focused.chromium.png|372| Changed|
|Breadcrumb Converged.divider type.default.chromium.png|374| Changed|
|Breadcrumb Converged.divider type.focused.chromium.png|374| Changed|
|Breadcrumb Converged.size.pressed.chromium.png|372| Changed|
|Breadcrumb Converged.divider type.pressed.chromium.png|374| Changed|

</details>




<div id="vrtCommentfluent-ui_VRT_Pipeline_v9"/>